pulsar-broker-common/src/main/java/org/apache/pulsar/common/configuration/PulsarConfigurationLoader.java

```diff
@@ -127,4 +130,34 @@ public static boolean isComplete(Object obj) throws IllegalArgumentException, Il
         return true;
     }
 
+    public static ServiceConfiguration convertFrom(PulsarConfiguration conf, boolean ignoreNonExistMember) throws RuntimeException {
+        try {
+            final ServiceConfiguration convertedConf = ServiceConfiguration.class.newInstance();
+            Field[] confFields = conf.getClass().getDeclaredFields();
+            Arrays.stream(confFields).forEach(confField -> {
+                try {
+                    Field convertedConfField = ServiceConfiguration.class.getDeclaredField(confField.getName());
+                    confField.setAccessible(true);
+                    convertedConfField.setAccessible(true);
+                    convertedConfField.set(convertedConf, confField.get(conf));
+                } catch (NoSuchFieldException e) {
+                    if (!ignoreNonExistMember) {
+                        throw new RuntimeException("Exception caused while converting configuration: " + e.getMessage());
```

Comment:
should we throw specific RuntimeException such as `IllegalArgumentException` in case of this validation? and also should we add comment about exception in method's java-doc section?